google/resource_google_service_account_key_test.go

```diff
@@ -0,0 +1,134 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+// Test that a service account key can be created and destroyed
+func TestAccGoogleServiceAccountKey_basic(t *testing.T) {
```

Comment:
Can you add t.Parallel() (and then a new line) to the beginning of each of these tests?